<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audio_wave_banks_document</class>
 <widget class="QWidget" name="audio_wave_banks_document">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Audio - Wave Banks</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QListWidget" name="wave_banks_list">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="wave_banks_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wave Banks</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="wave_banks_files">
     <item>
      <widget class="QLabel" name="files_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="files_list">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="files_controls">
       <item>
        <widget class="QToolButton" name="play_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="stop_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="files_saving">
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
